src/Logging.XUnit/XUnitLoggerExtensions.IMessageSink.cs

```diff
@@ -0,0 +1,342 @@
+﻿// Copyright (c) Martin Costello, 2018. All rights reserved.
```

Comment:
Looks like this part of the class is missing some test coverage.

Reply:
Well, it has identical coverage to its `ITestOutputHelper` counterpart. 🤷‍♀️ But I added a few tests in 144c55a5e4be472750cf803744f3279ababa7712 anyway.

Reply:
Fair enough - it was just the bot annotations that drew me to it 😃